### DISCUSSION TITLE
No admin after upgrade?

### DISCUSSION THREAD

**Discussion by kelvinburke** [TomBursch/kitchenowl#162] [Q&A] [answered]
Hi,

I was the admin user on my Kitchenowl server and could access the server admin settings, but after upgrading to v0.4.3 I cannot.
Have I somehow lost admin status during the upgrade or are the server admin settings missing / hidden in this version?

Can I re-instate my admin status by exec-ing into the backend docker container somehow?

I have attached an image of my ``/household/1/profile`` page
![image](https://github.com/TomBursch/kitchenowl/assets/13096142/9cc5a87c-749c-4d87-a49c-bd4031633198)

Thanks!

**Comment by kelvinburke** [comment, marked as answer]
I believe my user did somehow lose admin status in the upgrade, here are the steps that I used to fix it:

``cd`` in the backend server to the location of the docker volume provided to the backend container. In my case this was:
``volumes:
      - kitchenowl_data:/data``
So I found the ``kitchenowl_data`` directory

Find the ``database.db``
Optionally create a backup ``cp database.db database.db.bak``

Use ``sqlite3`` to edit the database:
``sqlite3 database.db``

(optionally) ``PRAGMA table_info(user);`` to see the column names of the ``user`` table

``SELECT * FROM user;``
Identify your user you want to make admin (name is the 1st column after ID), then
``UPDATE user SET admin=1 WHERE name IS "Kelvin";``
and replace ``Kelvin`` with whatever name you want

Check the last column is now ``1`` for the user you want, with ``SELECT * FROM user;``

Restart the container for good measure

**Reply by TomBursch** [reply]
Had no one else on the server admin rights? It seems like you weren't the only one who had that issue #157 

For any one else coming here and wondering how to do it the simple way:
Run `docker exec -it NAME_OF_THE_BACKEND_CONTAINER python manage.py`
This lets you interactively update users & households and mange the server.
